radsat-sk/operation/subsystems/RBattery.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file RBattery.h
+ * @date March 29, 2022
+ * @author Isaac Poirier (iap992)
+ */
+
+#ifndef RBATTERY_H_
+#define RBATTERY_H_
+
+#include <stdint.h>
+
+
+#endif /* RBATTERY_H_ */
+
+
+/***************************************************************************************************
+                                            DEFINITIONS
+***************************************************************************************************/
+
+uint8_t safeFlag;
```

Comment:
Why is there a public global? If you want others to be able to view the status of the safeFlag, then provide a getter public API function e.g. `batterySafe()`. Move this to be a private variable in the source file.